src/ctia/domain/entities.clj

```diff
@@ -34,44 +35,52 @@
                     (:end_time prev-valid-time)
                     time/default-expire-date)}})
 
+(defn default-realize
+  [type-name Model new-object id ident-map prev-object services]
+  (let [{{{:keys [get-in-config]} :ConfigService} :services} services
+        {:keys [login groups client-id]} ident-map
+        now (time/now)
+        with-base-fields (into new-object
+                               {:id id
+                                :type type-name
+                                :owner (or (:owner prev-object) login)
+                                :groups (or (:groups prev-object) groups)
+                                :schema_version schema-version
+                                :created (or (:created prev-object) now)
+                                :modified now
+                                :timestamp (or (:timestamp new-object)
+                                               (:timestamp prev-object)
```

Comment:
:created and :modified I understand, what's :timestamp is for?

Reply:
That's legacy, I do not have the context. It acts like created.
It's in [CTIM](https://github.com/threatgrid/ctim/blob/master/doc/tutorials/modeling-threat-intel-ctim.md#1112-optional-base-fields): "the time this object was created at, or last modified."
`created` and `modified` are specific to CTIA to distringuish both cases.